### PR TITLE
Added initializer for CreateFavouriteRequest from a given order, supplierId, and retailerId

### PR DESCRIPTION
**Pull request by mkmbayona**
Let me know if this is something we can use.

### CODE REVIEW

File: OrdermentumSDK/Models/Order.swift [hunk 1]
```diff
@@ -221,6 +221,31 @@ public struct CreateFavouriteRequest: Codable {
...
+    /**
+     * Creates an instance of CreateFavouriteRequest from a given order object, supplierId and retailerId
+     * Returns a CreateFavouriteRequest
+     */
+    public init(order:Order, retailerID: String, supplierID: String) {
+        //Initialize Array of Line Items
+        lineItems = []
+        
+        //Add Line Items
+        for cartProductObject in order.lineItems {
```
Comment (bstillitano) [inline OrdermentumSDK/Models/Order.swift]: Rename without reference to cart. The SDK has no concept of a cart. something like 'for lineItem in order.lineItems'

**Comment by bstillitano** [comment]
Can you please move this to an extension instead of an initialiaser. If you need help with this let me know

**Comment by mkmbayona** [comment]
Moving it to an extension would make it either a class or an instance method, is that right? I just want to know, what would be the advantages of making it into a method?

**Comment by bstillitano** [comment]
@mkmbayona no it would not become an instance method. I'll put a commit into this repo to show you what I mean

**Pull request merged**